build.sbt

```diff
@@ -202,7 +201,11 @@ lazy val commonJsLibSettings = lucumaScalaJsSettings ++ commonLibSettings ++ Seq
   dependencyOverrides ++= ScalaJSReact.value
 )
 
-lazy val commonES = Seq(
+lazy val commonModule = Seq(
+  scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
```

Comment:
I guess we should use ESModule kind

Reply:
MUnit in ScalaJS requires CommonJSModules (https://scalameta.org/munit/docs/getting-started.html#scalajs-setup). I will apply the setting only for `Test` though.

Reply:
Ok, this is not very important but I think it should work with ESModules too